.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```

Comment:
How about these sections: 
**Describe the feature**
**Describe the use case you need it for**
**Additional context**

I thought about maybe adding a checkbox for "Would you like to create a PR for it?", What do you think?

Reply:
I like it